runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java

```diff
@@ -109,7 +109,11 @@ public void onChatMessage(ChatMessage event)
 
 		final String message = event.getMessage();
 
-		if (message.startsWith("You successfully cook") || message.startsWith("You successfully bake") || message.startsWith("You manage to cook") || message.startsWith("You roast a"))
+		if (message.startsWith("You successfully cook") ||
+			message.startsWith("You successfully bake") ||
```

Comment:
Other multiline conditionals in this project are typically styled:

```
if (a
    || b
    || c)
{
    // ...
}
```

I'd  recommend using that style here as well.

Reply:
yeah, just changed it to that thinking it was the checkstyle error but thats not it. :/